vpr/src/route/partition_tree.h

```diff
@@ -25,6 +25,19 @@ inline Side operator!(const Side& rhs) {
     return Side(!size_t(rhs));
 }
 
+/** Part of a net. Used by \ref DecompNetlistRouter. */
+class VirtualNet {
+  public:
+    /** The net in question. */
+    ParentNetId net_id;
+    /** Clipped bounding box. This is needed to enable decomposing a net multiple times.
+     * Otherwise we would need a history of side types and cutlines to compute the bbox. */
+    t_bb clipped_bb;
+    /** Times decomposed -- don't decompose vnets too deeply or
```

Comment:
Explain a bit more.  If a net has been bipartitioned once (into two partitions) times_decomposed would be 1 (I think).  Should add that.